pages/apply.html

```diff
@@ -364,7 +314,7 @@
    <div class="usa-alert usa-alert-success">
       <div class="usa-alert-body">
         <h1 class="usa-alert-heading">Thank you!</h1>
-        <p class="usa-alert-text">Your form submission has been successfully submitted.</p>
+        <p class="usa-alert-text">Thank you for your application. We'll be in touch if there is a fit. For questions, please email us at join18f@gsa.gov.</p>
```

Comment:
@hestherchang I'll check whether the rest of this content is already covered on the confirmation screen.

In case it's not: applicants usually expect to hear back either way. Is it true that the Talent Team plans to reply only if they're interested in proceeding?